### PR TITLE
network-generator: also load drop-ins for networkd.conf from credentials

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: man/systemd.system-credentials.xml [hunk 1]
```diff
@@ -138,18 +138,20 @@
...
           <para>Configures network devices. Read by
-          <citerefentry><refentrytitle>systemd-network-generator.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>. These
-          credentials directly translate to a matching <filename>*.netdev</filename>,
-          <filename>*.link</filename> or <filename>*.network</filename> file. Example: the contents of a
-          credential <filename>network.link.50-foobar</filename> will be copied into a file
-          <filename>50-foobar.link</filename>. See
-          <citerefentry><refentrytitle>systemd.netdev</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+          <citerefentry><refentrytitle>systemd-network-generator.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>.
+          These credentials directly translate to a matching <filename>.conf</filename>,
+          <filename>*.link</filename>, <filename>*.netdev</filename>, or <filename>*.network</filename> file.
```
Comment (YHNdnzj) [inline man/systemd.system-credentials.xml]: Hmm, I think `*` doesn't make much sense here, and for `.conf` you didn't add that. So remove it from the rest config types for consistency?

**Review by YHNdnzj — APPROVED** [review]

**Comment by yuwata** [comment]
@YHNdnzj Thank you for the review. Now the sentences in systemd-network-generator(8) are copied to systemd.system-credentials(7).

**Comment by YHNdnzj** [comment]
Hmm, just realized that this should be mentioned in NEWS as well.

**Comment by yuwata** [comment]
> Hmm, just realized that this should be mentioned in NEWS as well.

I will update NEWS after merge.

**Pull request merged**